src/main/java/com/cleanroommc/groovyscript/compat/mods/chisel/Carving.java

```diff
@@ -95,6 +98,8 @@ public void removeVariation(String groupName, ItemStack item) {
                     .add("instead, edit the oredict via `oredict.remove('{}', {})`", groupName, GroovyScriptCodeConverter.asGroovyCode(item, false, false))
                     .error()
                     .post();
+        } catch (NullPointerException e) {
+            GroovyLog.get().exception("An exception occurred with chisel carving - possibly due to some other mod doing registry replacement.", e);
```

Comment:
Is there something the user can do to fix the crash? If no then it should just be an error. (And mention what the user is else supposed to do). Otherwise still display the fix message.

Reply:
Then the user is supposed to remove what they are currently doing? The message should say that,